src/shared/dissect-image.c

```diff
@@ -4070,3 +4069,29 @@ int verity_dissect_and_mount(
 
         return 0;
 }
+
+int get_common_dissect_directory(char **ret) {
+        _cleanup_free_ char *t = NULL;
+        int r;
+
+        /* A common location we mount dissected images to. The assumption is that everyone who uses this
```

Comment:
Should be double check that this assumption is true? IE, if the directory already exists and it's not empty, then bail out or fallback to the previous /tmp/xxxx? Just in case